### ISSUE TITLE
Multiple methods named 'cancel' found with mismatched result, parameter type or attribut

### ISSUE THREAD

**Issue by nitingohel** [rs/SDWebImage#2139]
### New Issue Checklist

* [x] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [x] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [x] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / tvos
 Platform Version        | e.g. 8.0
 SDWebImage Version      | e.g. 3.7.6
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 7.3
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

I just update new SDweb to my existing project. and got error in webcache.m **Multiple methods named 'cancel' found with mismatched result, parameter type or attribut**

![screen shot 2017-12-22 at 3 33 25 pm](https://user-images.githubusercontent.com/5137595/34294264-b82b5294-e72d-11e7-899f-eff6bc813ab2.png)

How to fix this.
